packages/client/src/encryption/RSAKeyPair.ts

```diff
@@ -41,59 +40,31 @@ async function exportCryptoKey(key: CryptoKey, { isPrivate = false } = {}): Prom
     return `-----BEGIN ${TYPE} KEY-----\n${exportedAsBase64}\n-----END ${TYPE} KEY-----\n`
 }
 
-// after RSAKeyPair is ready
-type InitializedRSAKeyPair = O.Overwrite<RSAKeyPair, {
-    privateKey: string
-    publicKey: string
-}>
-
 export class RSAKeyPair {
-    /**
-     * Creates a new instance + waits for ready.
-     * Convenience.
-     */
-    static async create(): Promise<RSAKeyPair> {
-        const pair = new RSAKeyPair()
-        await pair.onReady()
-        return pair
-    }
-
-    public privateKey: string | undefined
-    public publicKey: string | undefined
-    private _generateKeyPairPromise: Promise<void> | undefined
-
-    async onReady(): Promise<void> {
-        if (this.isReady()) { return undefined }
-        return this.generateKeyPair()
-    }
+    // the keys are in PEM format
+    public privateKey: string
+    public publicKey: string
```

Comment:
```suggestion
    public readonly privateKey: string
    public readonly publicKey: string
```
Can use `readonly` here?

Reply:
True, and actually can also be private